README.md

```diff
@@ -27,8 +24,8 @@
 
 #### Acknowledgements
 
-* Some parts of this sample application were inspired by the excellent 
-  [Java FX tutorial](http://code.makery.ch/library/javafx-8-tutorial/) by *Marco Jakob*. 
+* The application is based on a desktop address book application available on Github
```

Comment:
Good that you guys updated the acknowledgement. It'll be good to keep the original one on JavaFX too.